WolvenKit.App/Helpers/DocumentTools.cs

```diff
@@ -34,6 +35,12 @@ public DocumentTools(ILoggerService loggerService, Cr2WTools cr2wTools)
 
     # region appfile
 
+    public List<string> GetAllComponentNames(List<appearanceAppearanceDefinition> appearances)
```

Comment:
this should be a static function, or moved to RedDocumentViewMenuBar.xaml.cs as private func